pyccel/stdlib/gFTL_functions/Vector_extensions.inc

```diff
@@ -0,0 +1,64 @@
+#include <vector/header.inc>
+
+contains
+
+#define __IDENTITY(x) x
+#define __guard __vector_guard
+#include "parameters/T/copy_vector_T_to_internal_T.inc"
+#include "parameters/T/define_derived_macros.inc"
+
+#ifdef __T_IS_COMPLEX
+  #error "Complex types are not supported for min/max operations"
+#else
```

Comment:
Why do you need this? Complex comparison is available in Fortran I already added it to have sorted sets

Reply:
fixed!